lib/screens/alerts/alert.ex

```diff
@@ -1,3 +1,6 @@
+# This file has primarily v1 stuff, but there are a couple of utils we use in v2.
```

Comment:
Would it be useful to create a module `Screens.Alerts.V1.Alert` (or something similar) that has only functions that are used by v1? The benefit is that if/when we remove v1 references from code, we will have more confidence that what we are removing is not used by v2.